package.json

```diff
@@ -5,7 +5,7 @@
   "main": "scripts/deploy.ts",
   "scripts": {
     "test": "npx hardhat test test/HelloWorld.ts",
-    "deploy": "npx hardhat run --network goerli scripts/deploy.ts",
+    "deploy": "yarn run ts-node --files scripts/deploy.ts",
```

Comment:
We need to stick with one package manager. If you prefer yarn, I'm ok with using it. But in that case, please change all the scripts to use yarn.

Reply:
Ah yeah, my bad. Going with yarn then.

Reply:
done.